### ISSUE TITLE
Blocking issue on cloning: "Create REST server with NodeJS and Express" via Github Learning Lab

### ISSUE THREAD

**Issue by gianlucamarano23** [M3kH/hyf-course-nodejs-3#1]
Dear @M3kH ,

I'm currently trying to get access to your course repository: hyf-homework-nodejs-3, via Github Learning Lab. 

<img width="784" alt="Schermata 2020-04-13 alle 17 45 59" src="https://user-images.githubusercontent.com/30800402/79134924-d7693800-7dae-11ea-9b21-6e06aee2361d.png">
Github Learning Lab is correctly set up on my side and, likewise, repository permissions are granted.

When cloning the repository via the dedicated procedure, the following message error is displayed though:

<img width="1042" alt="Schermata 2020-04-13 alle 17 46 19" src="https://user-images.githubusercontent.com/30800402/79135083-15665c00-7daf-11ea-927b-0115eac9cbf9.png">

I tried to delete and reinstall the app on my profile and redo the sequence of actions to access the course, without success so far. 

Could you please advise on this?
Grateful in advance,

**Comment by M3kH** [comment]
@gianlucamarano23 @YeapGuy could you check if your issue is related to https://github.community/t5/GitHub-Learning-Lab/Stuck-Maybe-you-need-to-update-permissions-Here-s-how/td-p/28445 ?

**Comment by YeapGuy** [comment]
No, it is not that. I was able to take other courses. Perhaps _you_ should try that for your course repos?

**Comment by M3kH** [comment]
I did but got no luck in reproducing the issue. I will do more investigations.
